hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/container/balancer/MockedSCM.java

```diff
@@ -86,7 +86,7 @@ public MockedSCM(@Nonnull TestableCluster testableCluster) {
     }
   }
 
-  private void init(@Nonnull ContainerBalancerConfiguration balancerConfig, @Nonnull OzoneConfiguration ozoneCfg) {
+  void init(@Nonnull ContainerBalancerConfiguration balancerConfig, @Nonnull OzoneConfiguration ozoneCfg) {
```

Comment:
I'd prefer to save `init` as private method and add two public methods for `ContainerBalancerTask` creating (without running) to allow run it in another thread:
```
public @Nonnull ContainerBalancerTask createBalancerTask(
      @Nonnull ContainerBalancerConfiguration config
  ) {
    init(config, new OzoneConfiguration());
    return createBalancerTask(new ContainerBalancer(scm), config);
  }

  public @Nonnull ContainerBalancerTask createBalancerTask(
      @Nonnull ContainerBalancer containerBalancer,
      @Nonnull ContainerBalancerConfiguration config
  ) {
    return new ContainerBalancerTask(scm, 0, containerBalancer,
        containerBalancer.getMetrics(), config, false);
  }
```